cmd/gitops-server/cmd/cmd.go

```diff
@@ -40,6 +43,9 @@ type Options struct {
 	LoggingEnabled                bool
 	OIDC                          OIDCAuthenticationOptions
 	NotificationControllerAddress string
+	TLSCert                       string
```

Comment:
Just to match the flag and avoid confusion we could change `TLSCert` and `TLSKey` to `TLSCertFile` and `TLSKeyFile`